tests/tests.go

```diff
@@ -840,6 +840,78 @@ func getAggregatedTaskStatus(issue *api.Issue) (api.TaskStatus, error) {
 	return api.TaskDone, nil
 }
 
+// getNextTaskStatus gets the next task status that needs to be handle.
+// nolint
+func getNextTaskStatus(issue *api.Issue) (api.TaskStatus, error) {
```

Comment:
Is this something can be used for other functions such as approve all and wait all?

Reply:
Unfortunately not now, maybe we can do refactoring in the future.

Reply:
How about change `getAggregatedTaskStatus` to return `TaskRunning` and `TaskPending`?

Reply:
> How about changing `getAggregatedTaskStatus` to return `TaskRunning` and `TaskPending`? Then it is the same as this function and seems to break nothing.

Try it. Let's wait for the test.